### PR TITLE
wpanctl: Clean up the output of the `mfg` command.

### PR DESCRIPTION
**Pull request by darconeous**
The output from this command was previously a little janky and hard to parse. This commit fixes it.

### CODE REVIEW

File: src/wpanctl/tool-cmd-mfg.c [hunk 1]
```diff
@@ -37,6 +37,7 @@ int tool_cmd_mfg(int argc, char *argv[])
 	DBusConnection *connection = NULL;
 	DBusMessage *message = NULL;
 	DBusMessage *reply = NULL;
+   const char* result_cstr = NULL;
```
Comment (darconeous) [inline src/wpanctl/tool-cmd-mfg.c]: Wups. Will fix.

File: src/wpanctl/tool-cmd-mfg.c [hunk 2]
```diff
@@ -108,7 +108,12 @@ int tool_cmd_mfg(int argc, char *argv[])
 
 		dbus_message_iter_next(&iter);
 
-		dump_info_from_iter(stdout, &iter, 0, false, false);
+       if (dbus_message_iter_get_arg_type(&iter) == DBUS_TYPE_STRING) {
+           dbus_message_iter_get_basic(&iter, &result_cstr);
+           printf("%s", result_cstr);
+       } else {
+           dump_info_from_iter(stdout, &iter, 0, false, false);
+       }
```
Comment (abtink) [inline src/wpanctl/tool-cmd-mfg.c]: The indentation seems off here (maybe tabs vs spaces)...

Comment (darconeous) [inline src/wpanctl/tool-cmd-mfg.c]: Oops, will fix.
